src/preact/utils.js

```diff
@@ -30,12 +30,12 @@ function setRef(ref, value) {
 
 /**
  * Combines refs to pass into `ref` prop.
- * @param {import('preact').Ref<T>[]} refs
- * @return {function(T):void}
- * @template T
+ * @param {import('preact').Ref<*>[]} refs
+ * @return {import('preact').Ref<*>}
  */
 export function useMergeRefs(refs) {
   return useCallback(
+    /** @param {*} element */
```

Comment:
Why any instead of `Element` here?

Reply:
Ah. It works! Had issues before making it work with the param/return type.
Done now.